std-bits/table/src/main/java/org/enso/table/data/column/storage/DoubleStorage.java

```diff
@@ -1,35 +1,38 @@
 package org.enso.table.data.column.storage;
 
-import java.util.BitSet;
-import java.util.List;
-
 import org.enso.table.data.column.builder.object.Builder;
 import org.enso.table.data.column.builder.object.NumericBuilder;
+import org.enso.table.data.column.builder.object.StringBuilder;
+import org.enso.table.data.column.operation.CastProblemBuilder;
 import org.enso.table.data.column.operation.map.MapOpStorage;
 import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
 import org.enso.table.data.column.operation.map.UnaryMapOperation;
 import org.enso.table.data.column.operation.map.numeric.DoubleBooleanOp;
 import org.enso.table.data.column.operation.map.numeric.DoubleIsInOp;
 import org.enso.table.data.column.operation.map.numeric.DoubleNumericOp;
-import org.enso.table.data.column.storage.type.FloatType;
-import org.enso.table.data.column.storage.type.StorageType;
+import org.enso.table.data.column.storage.type.*;
 import org.enso.table.data.index.Index;
 import org.enso.table.data.mask.OrderMask;
 import org.enso.table.data.mask.SliceRange;
 import org.graalvm.polyglot.Value;
 
-/** A column containing floating point numbers. */
+import java.util.BitSet;
+import java.util.List;
+
+/**
+ * A column containing floating point numbers.
+ */
 public final class DoubleStorage extends NumericStorage<Double> {
   private final long[] data;
   private final BitSet isMissing;
   private final int size;
   private static final MapOpStorage<Double, DoubleStorage> ops = buildOps();
 
   /**
-   * @param data the underlying data
-   * @param size the number of items stored
+   * @param data      the underlying data
+   * @param size      the number of items stored
    * @param isMissing a bit set denoting at index {@code i} whether or not the value at index {@code
-   *     i} is missing.
+   *                  i} is missing.
```

Comment:
Why reformat like this?

Reply:
I did not notice this, sorry, probably my IDE messed things up.